website/scripts/build-static-content.js

```diff
@@ -40,6 +41,23 @@ module.exports = {
           } else if (query.resolution === undefined) {
             query.resolution = 'N/A';// « We set this to a string here so that the data type is always string.  We use N/A so folks can see there's no remediation and contribute if desired.
           }
+          if (query.tags) {
```

Comment:
What about if `!query.tags`?  I think we should set it to an empty array so that it's always the same data type.